extends Node2D

var text_main : String

# Called when the node enters the scene tree for the first time.
func _ready() -> void:
	randomize_text()
	$ColorRect/Label.text = text_main
	randomize_text()
	$ColorRect2/Label.text = text_main
	
	if not randi_range(0, 3):
		$ColorRect.queue_free()
	if not randi_range(0, 3):
		$ColorRect2.queue_free()
	
	$Timer.wait_time = randf_range(0.5, 20)
	$Timer.start()


# Called every frame. 'delta' is the elapsed time since the previous frame.
func _process(delta: float) -> void:
	pass

func randomize_text():
	var rolled_text = randi_range(0, 54)
	if rolled_text == 0:
		text_main = "download free REAL (real no clickbate!?1!?!!)"
	elif rolled_text == 1:
		text_main = "GTA 6 DONWLOAD NOW"
	elif rolled_text == 2:
		text_main = "free camel SEX real?"
	elif rolled_text == 3:
		text_main = "punjabi inshallah real no clickbate??! free sex"
	elif rolled_text == 4:
		text_main = "free RAM downlaod now (CLICK HERE for 100% real no fake free ram download 64GB!!1!)"
	elif rolled_text == 5:
		text_main = "real no fake minecraft thanos real? fortnite"
	elif rolled_text == 6:
		text_main = "ÙƒÙ…Ø°Ù‡Ù„ðŸ˜±ÙƒÙ… Ù„ÙƒÙ… Ø°Ù‡Ù„ÙƒÙ… Ù‡Ùˆ Ù…Ø°Ù‡"
	elif rolled_text == 7:
		text_main = "Ù‡Ø¬Ù…Ø§Øª 11 Ø³Ø¨ØªÙ…Ø¨Ø± ÙˆÙƒØ°Ù„Ùƒ Ø¨Ø§Ø³Ù… PETER GRIFFINÙ‡Ø¬Ù…Ø§Øª 11 Ø³Ø¨ØªÙ…Ø¨Ø± 2001 Ø£Ùˆ Ù‡Ø¬Ù…Ø§Øª PETER GRIFFIN JOIN ISIS ðŸ˜‚ðŸ˜‚ðŸ˜‚11 Ø£ÙŠÙ„ÙˆÙ„ØŒ ÙˆØªØ¹Ø±Ù Ø§Ø®ØªØµØ§Ø±Ù‹Ø§"
	elif rolled_text == 8:
		text_main = "Craft âœˆâœˆâœˆâœˆâœˆØ§Ù„ÙˆÙ„Ø§ÙŠØ§Øª Ø§Ù„Ù…ØªØ­Ø¯Ø© ÙÙŠ ÙŠÙˆÙ… Ø§Ù„Ø«Ù„Ø§Ø«Ø§Ø¡ Ø§Ù„Ù…ÙˆØ§Ù"
	elif rolled_text == 9:
		text_main = "free halal points halal gay camel sex hd 2019 free halal minecraft download Ù…Ø¬Ø§Ù†Ø§ Ø§Ù„Ø¬Ù…Ù„ Ø§Ù„Ø¥Ø¨Ø§Ø­ÙŠØ©ØŒ ØªØ­Ù…ÙŠÙ„ Ù…Ø§ÙŠÙ† ÙƒØ±Ø§ÙØª Ù…Ø¬Ø§Ù†Ø§ realØªØ­Ù…ÙŠÙ„ Ù„Ø¹Ù†Ø©"
	elif rolled_text == 10:
		text_main = "Ø§Ù„Ù„Ù‡ Ø¹Ù„Ù‰ Ø¥Ø³Ø±Ø§Ø¦ÙŠÙ„ ØªØ­Ù…ÙŠÙ„ Ù…Ø¬Ø§Ù†ÙŠ Ù…Ø¬Ø§Ù†Ø§ Ø§Ù„Ø¬Ù…Ù„ donlad trump sex? Ø§Ù„Ø¥Ø¨Ø§Ø­ÙŠØ©ØŒ ØªØ­Ù…ÙŠÙ„       Ù…Ø§ÙŠÙ† ÙƒØ±Ø§ÙØª no   fakeÙ…Ø¬Ø§Ù†Ø§ ØªØ­Ù…ÙŠ"
	elif rolled_text == 11:
		text_main = "Ø¥Ø³Ø±Ø§Ø¦ÙŠÙ„"
	elif rolled_text == 12:
		text_main = "ØªÙƒÙØ§ ÙŠØ§ Ø¬Ø±Ø§Ø­ ðŸ˜±ðŸ˜±ðŸ”¥ðŸ”¥ðŸ’¯ðŸ’¯â­•ï¸â­•ï¸â­•ï¸ðŸ‘‡ðŸ‘‡ðŸ‘‡"
	elif rolled_text == 13:
		text_main = "Download Grand Theft Auto 5 Android here free link Ù‚Ù… Ø¨ØªÙ†Ø²ÙŠÙ„ Grand Theft Auto 5 Android Ù‡Ù†Ø§ Ø§Ù„Ø±Ø§Ø¨Ø· Ø§Ù„Ù…Ø¬Ø§Ù†ÙŠ"
	elif rolled_text == 14:
		text_main = "Ù…Ø«Ù„ÙŠ Ø§Ù„ðŸ¤¨Ø¬Ù†Ø³ Ø§Ù„Ø¬Ù†Ø³ â‰ï¸ Ø§Ù„Ø¬Ù†Ø³ Ø§Ù„Ø¬Ù†Ø³ Ù…ðŸ¥³Ø«Ù„ÙŠ Ø§Ù„Ø¬Ù†Ø³ Ù…Ø«Ù„ÙŠØ§Ù„Ø¬Ù†Ø³ ðŸ¤‘ Ø§Ù„Ø¬Ù†Ø³ðŸ¥ºÙ…Ø«Ù„ÙŠØ§Ù„Ø¬Ù†Ø³Ø§Ù„Ø¬Ù†Ø³ Ù…ðŸ˜±Ø«Ù„ÙŠ Ø§Ù„Ø¬Ù†Ø³ Ù…Ø«Ù„ÙŠØ§Ù„Ø¬Ù†Ø³ ðŸ¤‘â‰ï¸ Ø§Ù„Ø¬Ù†Ø³"
	elif rolled_text == 15:
		text_main = "Ø§Ù„Ø¬Ù† ðŸ¤‘Ø³ Ù…Ø«Ù„ðŸ˜±ÙŠ Ø§Ù„Ø¬Ù†ðŸ¤¨Ø³ Ø§Ù„Ø¬Ù†Ø³ Ù…Ø«ðŸ™Ù„ÙŠ Ø§Ù„Ø¬Ù†Ø³ Ù…Ø«Ù„ÙŠ Ø§Ù„Ø¬Ù†Ø³ðŸ™ Ø§Ù„ðŸ¥ºØ¬Ù†Ø³ Ù…Ø«Ù„ÙŠ Ø§Ù„ðŸ¥³Ø¬Ù†Ø³ Ø§Ù„Ø¬Ù†Ø³ Ù…Ø«Ù„ÙŠ Ø§Ù„Ø¬Ù†Ø³ Ù…â‰ï¸ Ø§Ù„Ø¬Ù†Ø³ Ø§ðŸ¥³Ù„Ø¬Ù†Ø³ Ù…Ø«Ù„ÙŠ Ø§Ù„Ø¬Ù†Ø³"
	elif rolled_text == 16:
		text_main = "br7h HARAMðŸ‘ŽðŸ‘Ž ï­“ ï­” ï­• ï­– ï­—ï­˜ ï­™ï­š ï­› ï­ ï­ž ï­Ÿ ï­  ï­¡ ï­¢ ï­£ ï­¤ Ù…Ù† ÙØ¶Ù„Ùƒ ØŒ Ù„Ù… Ø£Ø±Ù‡Ù… Ù…Ù†Ø° ÙØªØ±Ø© Ø·ÙˆÙŠÙ„Ø© ØŒ Ù…Ù† ÙØ¶Ù„Ùƒ ÙÙ‚Ø· Ø¯Ø¹Ù†ÙŠ Ø£Ø°Ù‡Ø¨ ØŒ ÙˆØ§Ù„Ù‡Ø±Ø¨ Ø§Ù„ÙˆØ­ÙŠØ¯ Ù‡Ùˆ Ø§Ù„Ù…Ùˆ"
	elif rolled_text == 17:
		text_main = "infidels win? windows vista ultimate sp2 iso free download no pork no haram no virus gta 4 ps3 unboxing allah 2008"
	elif rolled_text == 18:
		text_main = "Bruh fart Ù…Ù„Ø­Ù…Ø© Ø¨Ø±ÙˆØ© Ø¶Ø±Ø·Ø© Ø§Ù„Ø¹Ø±Ø¨ÙŠØ© infidels win??? ðŸ˜ðŸ˜ðŸ˜ðŸ˜"
	elif rolled_text == 19:
		text_main = "B u ruh porkchop is not Allah approve ðŸ˜ ðŸ˜ ðŸ˜  ðŸ–•ðŸ–• ØªÙ…Ø³ØªØ¨ Ùˆ Ù†Ø±ØªØ§ Ùˆ ÙƒØ§ÙŠÙ† Ù„ÙŠ ÙƒÙˆØ¨ ÙŠØ±Ù‚ØµÙˆ"
	elif rolled_text == 20:
		text_main = "www.free-camelfart_sex.com"
	elif rolled_text == 21:
		text_main = "Pan arabism be like"
	elif rolled_text == 22:
		text_main = "TROLL Ø¥Ø· ØµÙ†Ø£ÙˆØ¥Ø¯ØŒ Ø±Ø£Ø¥Ù†Ø¥Ø¯ØŒ Ø£Ù†Ø¯ Ù‡Ø£Ø¥Ù„Ø¥Ø¯ Ø·Ù‡Ø¥ ØµØ£Ù…Ø¥ Ù…Ø£Ø±Ù†Ø¥Ù†Ø¬. FUNNY!!"
	elif rolled_text == 23:
		text_main = "Ø§Ù„ØªÙƒÙ†ÙˆÙ„ÙˆØ¬ÙŠØ§ ØªØªØ­Ø³Ù†"
	elif rolled_text == 24:
		text_main = "ehhe funny arabic sunnah time!!!!"
	elif rolled_text == 25:
		text_main = "Airplane danceðŸ¤£"
	elif rolled_text == 26:
		text_main = "kogan portable air conditionerÙ…ÙƒÙŠÙ Ø§Ù„Ù‡ÙˆØ§Ø¡ Ø§Ù„Ù…Ø­Ù…ÙˆÙ„Ù…ÙƒÙŠÙ Ø§Ù„Ù‡ÙˆØ§Ø¡ Ø§Ù„Ù…Ø­Ù…ÙˆÙ„"
	elif rolled_text == 27:
		text_main = "HARAM OBV SPRUNKI DOWNLOAD ØªØ­Ù…ÙŠÙ„ ÙƒØªØ§Ø¨ Ø­Ø±Ø§Ù… OBV SPRUNKI"
	elif rolled_text == 28:
		text_main = "Halal bobjiâ­•â­•ðŸ˜¸ðŸ˜¸ðŸ˜¸ðŸ§¤ðŸ‘¨ðŸ¿â€ðŸŒ¾ðŸ‘¨ðŸ‘¨ðŸ¿â€ðŸŒ¾ðŸ‘¨ðŸ¿â€ðŸŒ¾"
	elif rolled_text == 29:
		text_main = "Vanceshallah"
	elif rolled_text == 30:
		text_main = "Ø³Ø¨Ø±Ø¯ÙŠØª Ù„Ø§Ø®Ø¨Ø§Ø± Ù…ØµØ± Ùˆ Ø§Ø®Ø¨Ø§Ø± Ø¹Ø§Ù„Ù…ÙŠØ©"
	elif rolled_text == 31:
		text_main = "Hottest Saudi mixtape by Bader200306 ðŸ’¯ðŸ”¥"
	elif rolled_text == 32:
		text_main = "ã€ï»¿ï¼¨ï¼¤ï¼´ï¼¶ã€‘Ø§Ù„Ø¨ÙŠØ§Ù† Ø§Ù„Ø³ÙŠØ§Ø³ÙŠ Ù„Ù„Ø£Ù…Ø© Ù„Ù„ÙˆÙ„Ø§ÙŠØ§Øª Ø§Ù„Ù…ØªØ­Ø¯Ø© Ð¡Ð¢Ð˜Ð’Ð•Ð ÐšÐžÐ›Ð‘Ð•Ð Ð¢.wmv"
	elif rolled_text == 33:
		text_main = "Ø¨Ù†ÙŠ Ù‚Ù…Ø« Ø§ ÙØ¸ Ø²Ù…ÙŠÙƒØ« Full hd 4k"
	elif rolled_text == 34:
		text_main = "ðŸŒœðŸ™ðŸ™ðŸ™Ø§Ù„Ø³Ø­ÙˆØ± Ù‡Ù†Ø§ Ù…Ø§Ø´Ø§Ø¡Ø§Ù„Ù„Ù‡ðŸ™"
	elif rolled_text == 35:
		text_main = "Ø¢Ø®Ø± Ø¥Ù†Ø³Ø§Ù† Ù…Ù†ØªØµØ¨ Ø¹Ù„Ù‰ Ù‚ÙŠØ¯ Ø§Ù„Ø­ÙŠØ§Ø© ÙÙŠ Ø£ÙˆÙƒØ±Ø§Ù†ÙŠØ§"
	elif rolled_text == 36:
		text_main = "signa momento"
	elif rolled_text == 37:
		text_main = "Ø¨Ù†ÙŠÙ† Ù„Ù…ÙŠÙ„Ù†Ù† Ø´Ù‡Ù‡Ù‡Ù‡Ù‡Ù‡Ù‡Ù‡Ù‡Ù‡Ù‡Ù‡ðŸ¥¶ðŸ¥¶ðŸ¥µðŸ¥µ"
	elif rolled_text == 38:
		text_main = "Homander ÙŠÙ„Ù…Ø³ Ø¨Ø·Ù† Ø´Ø®Øµ Ù„Ø§ Ø§Ø¹Ø±ÙÙ‡ Ø­ØªÙ‰ Ù„Ø§ Ø§Ø¹Ø±ÙÙ‡ Ù„Ù… Ø§Ø´Ø§Ù‡Ø¯ Ø§Ù„Ù…Ø³Ù„Ø³Ù„"
	elif rolled_text == 39:
		text_main = "Minecraft xD pigs nuke die and villagers haram subhan allah halal asylum punjabi"
	elif rolled_text == 40:
		text_main = "mascubeheyahallah minecraf halal asylum haram"
	elif rolled_text == 41:
		text_main = "â˜ªï¸ðŸ¤²ðŸ“¿ðŸ•‹ðŸ•Œ bery sreios adwise 4 theez Ramadan timez....... thankz to our Brozer parVin DiesEl-Deen (ÙŠØµÙ„Ø­ Ø§Ù„Ù„Ù‡ Ø¯Ù…Ø§ØºÙ‡) ðŸ™ðŸ’¥ðŸ’¥ðŸ’¥ðŸ’¥"
	elif rolled_text == 42:
		text_main = "Ø¹Ù†Ø¯Ù…Ø§ ØªÙ‚ÙˆÙ„ Ø²ÙˆØ¬ØªÙƒ Ø§Ù„Ø­Ø§Ø¯ÙŠØ© Ø¹Ø´Ø± Ø¥Ù†Ù‡Ø§ ØªÙƒØ±Ù‡ Ø§Ù„Ù„Ù‡"
	elif rolled_text == 43:
		text_main = "áƒ›áƒáƒ›áƒ”áƒœáƒ¢áƒ˜ áƒ áƒáƒªáƒ áƒ›áƒ—áƒšáƒ˜áƒáƒœáƒ˜ áƒ’áƒ£áƒœáƒ“áƒ˜ áƒ¨áƒ”áƒ›áƒáƒ’áƒ˜áƒ”áƒ áƒ—áƒ“áƒ, áƒ›áƒáƒ’áƒ áƒáƒ› áƒªáƒ£áƒ“áƒ˜ áƒ•áƒáƒ˜áƒ¤áƒáƒ˜ áƒ’áƒáƒ¥áƒ•áƒ§ðŸ¤£ðŸ¤£ðŸ˜‚ðŸ˜‚"
	elif rolled_text == 44:
		text_main = "Ø¹Ù†Ø¯Ù…Ø§ ÙŠØ¹ÙŠØ¯Ùƒ Ø§Ù„Ù…Ø¹Ø§Ù„Ø¬ Ù„Ù„Ø­ÙŠØ§Ø©ØŒ Ù„ÙƒÙ†Ù‡ ÙŠÙ†Ø³Ù‰ Ø¹Ù„Ø§Ø¬ Ø°Ø±Ø§Ø¹Ùƒ. Ø¹Ù†Ø¯Ù…Ø§ Ù…Ù† Ø§Ù„Ù…ÙØªØ±Ø¶ Ø£Ù† ØªØ­ÙƒÙ… Ø§Ù„Ø¬Ø­ÙŠÙ…ØŒ Ù„ÙƒÙ† Ø§Ù„Ø¹Ø±Ø´ Ù…Ø±ÙŠØ­ Ø¬Ø¯Ù‹Ø§."
	elif rolled_text == 45:
		text_main = "Those who know ÙÙŠØ¯ÙŠÙˆ Ù‚ØµÙŠØ± Ø¬Ù…ÙŠÙ„ Ø¬Ø¯Ø§ rick astley!!"
	elif rolled_text == 46:
		text_main = "ÛÙ… Ù†Û’ Ù…ÛŒÚ¯Ù†Ø³ Ú©ÛŒ Ø¯Ù†ÛŒØ§ Ú©ÛŒ Ø¢Ø¨Ø§Ø¯ÛŒ Ù¾Ø± ØºÙ„Ø¨Û Ø­Ø§ØµÙ„ Ú©ÛŒØ§ ÛÛ’Û”Halal download no virus"
	elif rolled_text == 47:
		text_main = "habibi hamood roblox"
	elif rolled_text == 48:
		text_main = "IN DA CLUB Ø£Ø·Ø±Ù ÙÙŠØ¯ÙŠÙˆ Ø¹Ù„Ù‰ Ø§Ù„Ø¥Ù†ØªØ±Ù†Øª 2005 Ø¹Ù„Ù‰ Ø´Ø¨ÙƒØ© Ø§Ù„Ø¥Ù†ØªØ±Ù†ØªðŸ˜‚ðŸ˜‚ðŸ˜‚ðŸ˜‚ðŸ˜‚ðŸ˜‚"
	elif rolled_text == 49:
		text_main = "Mile Morales Ù…Ù„Ù Python Ø§Ù„Ù‚Ø§Ø¨Ù„ Ù„Ù„ØªÙ†ÙÙŠØ° Ø¨Ø§Ù„Ù„ØºØ©MP4 Ø§Ù„Ø¨Ø¹Ø«ÙŠØ© Ù‡Ùˆ Ù…ÙƒØ§Ù† Ø±Ø§Ø¦Ø¹ Eddie Murphy Ø¹Ø±Ø¨ÙŠ Ù…Ø¶Ø­Ùƒ FOLLOW RULES ðŸ—£ðŸ’”ðŸ’”ðŸ’” Ba'athism GTA 7 pdf leaked"
	elif rolled_text == 50:
		text_main = "ðŸ˜³ðŸ˜³sponshy bobes Ø³ÙÙ†Ø¬Ù‡ Ø§Ù„Ø¨ÙˆØ¨ÙŠ ðŸ˜‚ðŸ˜³"
	elif rolled_text == 51:
		text_main = "War crime 100% serbia halal Ø²ÙŠ Ù‚Ù…Ø«Ù…Ø«Ø¸ Ù„Ù…Ø§Ù…Ø²Ù‚ Ø§Ù…Ù…Ù„ ðŸ˜‚ðŸ˜‚ðŸ˜‚ðŸ˜‚ðŸ‘"
	elif rolled_text == 52:
		text_main = "Ø­Ø± sprunki download Ù„Ø§ ÙŠÙˆØ¬Ø¯ ÙÙŠØ±ÙˆØ³ pokemon legends arceus download ðŸ†ðŸ…ðŸ«"
	elif rolled_text == 53:
		text_main = "mineraft free donwload no virus 100% working"
	elif rolled_text == 54:
		text_main = "bottom Text"

func _on_timer_timeout() -> void:
	queue_free()
